base/utils/R/convert.input.R

```diff
@@ -636,7 +636,7 @@ convert.input <-
     }
     
     # post zipped file to Brown Dog
-    html <- RCurl::postForm(url, fileData = RCurl::fileUpload(zipfile), .opts = curloptions)
+    html <- curl::form_file(url, type = "___")
```

Comment:
It looks like the recommended pattern for form posting is to build up the multipart request inside the handle, then pass the whole handle as a fetch. See [this example](https://cran.r-project.org/web/packages/curl/vignettes/intro.html#posting_forms) in the curl vignette.

Reply:
@infotroph thanks for the review ,I'll start working on the other files .